fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1354,6 +1376,7 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 				metricPromiseStream->send(
 				    std::make_pair(ROCKSDB_READVALUE_GET_HISTOGRAM.toString(), timer_monotonic() - dbGetBeginTime));
 			}
+			sharedState->readLatency[threadIndex]->addMeasurement(timer_monotonic() - readBeginTime);
```

Comment:
Can you store the timer_monotonic in a variable and use it in these two places? Just to avoid the cost.

Reply:
Done.